c_api/src/c_api.rs

```diff
@@ -428,6 +429,22 @@ pub unsafe extern "C" fn c2pa_string_free(s: *mut c_char) {
     }
 }
 
+// Run CAWG post-validation - this is async and requires a runtime.
+fn post_validate(result: Result<C2paReader, c2pa::Error>) -> Result<C2paReader, c2pa::Error> {
+    match result {
+        Ok(mut reader) => {
+            let runtime = match Runtime::new() {
+                Ok(runtime) => runtime,
+                Err(err) => return Err(c2pa::Error::OtherError(Box::new(err))),
+            };
+            match runtime.block_on(reader.post_validate_async(&CawgValidator {})) {
```

Comment:
Does the CAWG validation pass by default if there is no CAWG data?

Reply:
Yes.